Notebooks/QC/index.ipynb

```diff
@@ -94,26 +88,25 @@
     "\n",
     "    cd /home/manager/pathogen-informatics-training/Notebooks/QC/\n",
     "    \n",
-    "Now you can follow the instructions in the tutorial from here.\n",
+    "You should set up a Conda environment to install software. Now you can follow the instructions in the tutorial from here.\n",
     "\n",
     "## Let’s get started!\n",
     "This tutorial assumes that you have samtools, bcftools, bwa, Picard tools and Kraken installed on your computer. For download and installation instructions, please see:\n",
     "\n",
-    "* The [samtools website](http://samtools.sourceforge.net/)\n",
-    "* The [bcftools website](http://www.htslib.org/download/)\n",
-    "* The [bwa GitHub page](https://github.com/lh3/bwa)\n",
-    "* The [Picard tools GitHub page](https://broadinstitute.github.io/picard/)\n",
-    "* The [Kraken website](https://www.ebi.ac.uk/research/enright/software/kraken)\n",
```

Comment:
Better: Briefly describe how to set up a conda environment and then just list the commands that need to be run (`conda install samtools` etc). No need to refer to individual websites.